schemas/dispec-driven/templates/dependencies.md

```
@@ -0,0 +1,37 @@
+### Wave 3
+<!-- Tasks that depend on Wave 1 or Wave 2 -->
+
+## Float / Slack
+
+| Task ID | Float | Notes |
+|---------|-------|-------|
+| <!-- task --> | <!-- amount --> | <!-- can be delayed without affecting timeline --> |
+
+## Text DAG
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)